tests/phpunit/integration/tests/REST_API/Stories_Controller.php

```diff
@@ -427,7 +465,7 @@ public function test_filter_posts_by_author_display_names() {
 		$response = rest_get_server()->dispatch( $request );
 		$results  = wp_list_pluck( $response->get_data(), 'author' );
 
-		$this->assertSame(
+		$this->assertEquals(
```

Comment:
I don't understand why this was changed again. `assertSame` should be used wherever possible for type checks